suite-native/storage/src/transforms/utils.ts

```diff
@@ -8,15 +8,11 @@ export const selectDeviceStatesNotRemembered = (state: DeviceRootState) => {
     );
 };
 
-export const filterObjectKeys = <O extends Record<string, any>>(
+export const filterKeysByPartialMatch = <O extends Record<string, unknown>>(
```

Comment:
Looks much better 👍 but please update the unit test.

Reply:
done